src/Symfony/Component/Messenger/DataCollector/MessengerDataCollector.php

```diff
@@ -79,38 +80,25 @@ private function collectMessage(string $busName, array $tracedMessage)
         $debugRepresentation = array(
             'bus' => $busName,
             'message' => array(
-                'type' => \get_class($message),
-                'object' => $this->cloneVar($message),
+                'type' => new ClassStub(\get_class($message)),
+                'value' => $message,
             ),
         );
 
         if (array_key_exists('result', $tracedMessage)) {
             $result = $tracedMessage['result'];
-
-            if (\is_object($result)) {
```

Comment:
I don't feel like we need all these conditions and formats... let me know if there was a reason.